### PR TITLE
Fixes for integration tests php81 compatibility

### PR DESCRIPTION
**Pull request by SilinMykola**
### Description (*)
Fixes for integration tests php81 compatibility

### Related Issues (if relevant)
1. Fixes magento/magento2#<issue_number>

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml [hunk 1]
```diff
@@ -53,7 +53,7 @@ if ($isField) {
 
     <?php if (!$element->getNoContainer()) : ?>
         <fieldset class="<?=  $block->escapeHtmlAttr($cssClass) ?>" id="<?=  $block->escapeHtmlAttr($id) ?>">
-        <?php if (strlen($element->getBeforeElementHtml())) : ?>
+        <?php if (strlen(is_string($element->getBeforeElementHtml())) ? $element->getBeforeElementHtml() : '') : ?>
```
Comment (Den4ik) [inline app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml]: I believe that we can move this logic into `getBeforeElementHtml` method that prevent twice call
```
$beforeHtml = $this->getData('before_element_html');
return is_string($beforeHtml) ? $beforeHtml : '';
```

**Review by Den4ik — CHANGES_REQUESTED** [review]
Hi @SilinMykola. Thanks for the work on the updates.
Please look at my comment below.

File: app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml (hunk 1 above)
Comment (SilinMykola) [reply app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml]: @Den4ik Hi! I think that it's a bad idea to change return type for api Abstract class.

**Comment by xmav** [comment]
@magento run all tests

**Comment by xmav** [comment]
@magento run Static Tests, Functional Tests EE

**Comment by xmav** [comment]
@magento run Static Tests, Functional Tests EE

**Comment by xmav** [comment]
@magento run Static Tests

**Comment by Den4ik** [comment]
Hi @SilinMykola 
Thanks for updates. Are looking to failed tests?

**Comment by SilinMykola** [comment]
@magento run all tests

**Comment by SilinMykola** [comment]
@magento run Static Tests

**Comment by SilinMykola** [comment]
@magento run Static Tests

**Comment by SilinMykola** [comment]
@magento run Static Tests

**Comment by andrewbess** [comment]
@mageto run Static Tests, Semantic Version Checker

File: app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml [hunk 2]
```diff
@@ -51,12 +51,12 @@ if ($isField) {
             data-role="<?=  $block->escapeHtmlAttr($id) ?>-content">
 <?php endif; ?>
 
-    <?php if (!$element->getNoContainer()) : ?>
+    <?php if (!$element->getNoContainer()): ?>
         <fieldset class="<?=  $block->escapeHtmlAttr($cssClass) ?>" id="<?=  $block->escapeHtmlAttr($id) ?>">
-        <?php if (strlen($element->getBeforeElementHtml())) : ?>
+        <?php if (strlen(is_string($element->getBeforeElementHtml())) ? $element->getBeforeElementHtml() : ''): ?>
```
Comment (andrewbess) [inline app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml]: I think the condition should look like proposed below
```suggestion
        <?php if (is_string($element->getBeforeElementHtml()) && strlen($element->getBeforeElementHtml()): ?>
```
@SilinMykola how do you think?

File: lib/internal/Magento/Framework/Data/Form/Element/AbstractElement.php [hunk 3]
```diff
@@ -62,8 +62,6 @@ abstract class AbstractElement extends AbstractForm
     protected $_escaper;
 
     /**
-     * Lock html attribute
-     *
```
Comment (andrewbess) [inline lib/internal/Magento/Framework/Data/Form/Element/AbstractElement.php]: @SilinMykola perhaps this file should be reverted

**Review by andrewbess — CHANGES_REQUESTED** [review]
Hello @SilinMykola 
Thank you for your contribution.
Could you please check my comments?
Thank you in advance.

File: app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml (hunk 2 above)
Comment (SilinMykola) [reply app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml]: @andrewbess Hi! Thanks for your advice, I made needed changes.

File: lib/internal/Magento/Framework/Data/Form/Element/AbstractElement.php (hunk 3 above)
Comment (SilinMykola) [reply lib/internal/Magento/Framework/Data/Form/Element/AbstractElement.php]: @andrewbess Hi! Thanks for your advice, I made needed changes.

**Comment by xmav** [comment]
@magento run Static Tests, Semantic Version Checker

**Comment by xmav** [comment]
@magento run Static Tests, Semantic Version Checker

**Comment by andrewbess** [comment]
@magento run Static Tests, Semantic Version Checker

**Comment by andrewbess** [comment]
@magento run Static Tests, Semantic Version Checker

**Review by andrewbess — APPROVED** [review]
Hello @SilinMykola 
Thank you for your work here
Approved ✔️

**Review by Den4ik — APPROVED** [review]

**Review by eduard13 — APPROVED** [review]

**Comment by xmav** [comment]
pending delivery https://github.com/magento-commerce/magento2ce/pull/7255

**Pull request merged**